repos/spack_repo/builtin/packages/express/package.py

```diff
@@ -56,5 +57,15 @@ def patch(self):
                 "%s" % self.spec["bamtools"].libs,
             )
 
+            # Dont try and compile against specific file
+            edit.filter(
+                r"\${PROTOBUF_LIB_DIR}/" "libprotobuf.a", "%s" % self.spec["protobuf"].libs
```

Comment:
```suggestion
                r"\${PROTOBUF_LIB_DIR}/libprotobuf.a", "%s" % self.spec["protobuf"].libs
```
The string concatenation in this line is really difficult to read as-is

You could also use the `string=True` argument to avoid needing to escape regex symbols.